docs/mine/lotus/miner-setup.md

```diff
@@ -102,6 +102,7 @@ lotus-miner init --owner=<bls address>  --no-local-storage
 - The `--no-local-storage` flag is used so that we can later configure [specific locations for storage](custom-storage-layout.md). This is optional but recommended.
 - The init process will download over **100GiB of initialization parameters** to /var/tmp/filecoin-proof-parameters. Make sure there is space or set `FIL_PROOFS_PARAMETER_CACHE` to somewhere else. We additionally recommend that this corresponds to an NVMe drive, as reading and verifying the proofs affects how quickly a miner becomes fully booted every time it is started.
 - The Lotus Miner configuration folder is created at `~/.lotusminer/` or `$LOTUS_MINER_PATH` if set.
+- The _owner address_ is used to create the miner on the chain (by submitting a message), upon which the miner receives a unique miner ID which is both human-readable and can act as an alias to the ower-address.
```

Comment:
Is it the owner or the worker address?

Reply:
From below it would seem to be the worker address.

Reply:
both owner address and worker address are required for init a miner. (if one doesnt specify the worker address, lotus creates one for the miner)

Reply:
Not really, that is only with the --create-worker flag. Otherwise the owner one is used.

Reply:
- The miner ID isn't an alias to the owner-address...it _is_ the address of the miner actor.
- If a worker address isn't specified when initing a miner, the owner address will be used as both owner and worker.
- Typo in "ower-address".